cmd/skuba/cluster/check.go

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2020 SUSE LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package cluster
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/SUSE/skuba/internal/pkg/skuba/kubeadm"
+	"github.com/SUSE/skuba/internal/pkg/skuba/kubernetes"
+	"github.com/SUSE/skuba/pkg/skuba"
+	"github.com/SUSE/skuba/pkg/skuba/actions/cluster/upgrade"
+	"github.com/rikatz/kubepug/lib"
+	"github.com/rikatz/kubepug/pkg/formatter"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/util/version"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/klog"
+)
+
+type checkOptions struct {
+	K8sVersion      string
+	ForceDownload   bool
+	APIWalk         bool
+	SwaggerDir      string
+	ShowDescription bool
+}
+
+var (
+	format    string
+	filename  string
+	inputFile string
+)
+
+// newUpgradeCheckCmd creates a new `skuba check` cobra command
```

Comment:
NIT and can be merged without fixing as it is just a comment, but I would ether write the entire command aka `skuba cluster upgrade check` or just the fact that it's a new cobra command. though honestly it is easy to see it's a new cobra command, would probably do something like...
```suggestion
// newUpgradeCheckCmd creates a new `check` cobra command that tests for deprecated APIs
```

Reply:
@chentex please take this suggestion from @Klaven